Resources/Public/IconFonts/style.css

```diff
@@ -1,438 +1,442 @@
 @font-face {
   font-family: 'icons';
   src:
-    url('fonts/icons.ttf?ifwz8v') format('truetype'),
-    url('fonts/icons.woff?ifwz8v') format('woff'),
-    url('fonts/icons.svg?ifwz8v#icons') format('svg');
+    url('fonts/icons.ttf?fpk20a') format('truetype'),
+    url('fonts/icons.woff?fpk20a') format('woff'),
+    url('fonts/icons.svg?fpk20a#icons') format('svg');
   font-weight: normal;
   font-style: normal;
 }
 
-.icons {
```

Comment:
You should use class `.icons` as a CSS selector  instead of an attribute selector, otherwise, we will have a conflict with some content elements like a `icon-text-btn`.
https://github.com/t3kit/theme_t3kit/blob/master/Resources/Public/IconFonts/README.md

Thanks